code/modules/clothing/spacesuits/hardsuit.dm

```diff
@@ -382,16 +384,21 @@
 	alt_desc = "An elite version of the syndicate hardsuit, with improved armour and fireproofing. It is in combat mode."
 	icon_state = "hardsuit0-syndielite"
 	hardsuit_type = "syndielite"
-	helmettype = /obj/item/clothing/head/helmet/space/hardsuit/syndi/elite
+	helmettype = /obj/item/clothing/head/helmet/space/hardsuit/syndi/elite/debug
 	armor = list("melee" = 35, "bullet" = 40, "laser" = 35, "energy" = 20, "bomb" = 25, "bio" = 30, "rad" = 20, "fire" = 60, "acid" = 0, "wound" = 40)
 	heat_protection = CHEST|GROIN|LEGS|FEET|ARMS|HANDS
 	max_heat_protection_temperature = FIRE_IMMUNITY_MAX_TEMP_PROTECT
 	resistance_flags = FIRE_PROOF | ACID_PROOF
 	mutantrace_variation = STYLE_DIGITIGRADE|STYLE_SNEK_TAURIC|STYLE_PAW_TAURIC
 
 /obj/item/clothing/suit/space/hardsuit/syndi/elite/debug
+	name = "administrator hardsuit"
 	helmettype = /obj/item/clothing/head/helmet/space/hardsuit/syndi/elite/debug
 	slowdown = 0
+	armor = list("melee" = 99, "bullet" = 95, "laser" = 95, "energy" = 95, "bomb" = 95, "bio" = 100, "rad" = 100, "fire" = 100, "acid" = 100, "wound" = 100)
+
+/obj/item/clothing/suit/space/hardsuit/syndi/elite/debug/Initialize()
```

Comment:
![image](https://user-images.githubusercontent.com/49745305/218509640-ab202471-064e-4e54-908e-cff4d0982a14.png)

Reply:
what's the issue exactly? I was able to compile it just fine

Reply:
Run a debug. Compiling won't show the error, you need to run the game and verify your work.

Reply:
No, I Was compiling it AND tested it in game @Malerus  I had no issues in game

Reply:
You forgot the . =..()

Reply:
yeah done then